documentation/asciidoc/stories/prometheus_integration.adoc

```diff
@@ -0,0 +1,77 @@
+## Prometheus integration
+This chapter is about sending Infinispan statistics to Prometheus in an operator managed environment running on Openshift.
+
+#### Prerequisites
+1. Access to an Openshift console
+2. A running Prometheus server, installed via Operator (see )
+3. A running Infinispan cluster
+
+#### Deploy the service monitor
+A service monitor is a customer resource (ServiceMonitor) that represent the application to be monitored.
+The Prometheus Operator, when a ServiceMonitor cr is deployed to the system, genetares all the configuration needed to scrap the application.
+The information needed to define a ServiceMonitor are:
+
+1. how to select the workload to scrap. This will be represented by a selector item;
+2. how to connect to selected workload. This will be represented by an endpoint item.
+
+The ServiceMonitor template to populate is the following:
```

Comment:
Is this example intentionally "compressed"? It's missing `kind` and `metadata`.

Reply:
bad copy/paste. thanks!

Reply:
done